applications/common/backend/src/main/resources/libs/kyanite/common/components/button/dialog/.content.json

```diff
@@ -16,6 +16,12 @@
         "label": "Hide label",
         "description": "Use to describe link without showing the label. Can be useful e.g. to describe link with an icon"
       },
+      "buttonId": {
+        "sling:resourceType": "wcm/dialogs/components/textfield",
+        "name": "buttonId",
+        "label": "Button ID",
+        "description": "ID used for GTM/GA4/etc. purposes"
```

Comment:
In case of potential new future names (acronyms) for this feature, maybe a more general "ID used for Google Analytics purposes" would fit better? 💭